test/ExecutionFramework.h

```diff
@@ -275,6 +275,7 @@ class ExecutionFramework
 	solidity::frontend::RevertStrings m_revertStrings = solidity::frontend::RevertStrings::Default;
 	solidity::frontend::OptimiserSettings m_optimiserSettings = solidity::frontend::OptimiserSettings::minimal();
 	bool m_showMessages = false;
+	bool m_showMetadata = false;
```

Comment:
Well, since the ``ExecutionFramework`` can't do this, but everything is done in ``SolidityExecutionFramework`` instead, we should probably move the member there and initialize it in that constructor, shouldn't we?